src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry during our GVC I think I said to delete this, but actually I think we need to keep this file.

The index.html and script and style can stay deleted.